manual/sphinx/conf.py

```diff
@@ -30,16 +30,15 @@
 import subprocess
 import sys
 
-# Disable breathe
-has_breathe = False
+sys.path.append("../../tools/pylib")
 
 # Are we running on readthedocs?
 on_readthedocs = os.environ.get("READTHEDOCS") == "True"
 
 # readthedocs currently runs out of memory if we actually dare to try to do this
-if has_breathe and not on_readthedocs:
+if has_breathe:
     # Run doxygen to generate the XML sources
-    subprocess.call("cd ../doxygen; doxygen Doxyfile", shell=True)
+    subprocess.call("cd ../doxygen; doxygen Doxyfile_readthedocs", shell=True)
```

Comment:
The readthedocs environment should persist between repeat runs (i.e. when generating the different output types) so it should be possible to only do this once per build rather than once per format. I'll just record here that you've timed this and noted that this call is a relatively small fraction of the time.